app/javascript/pages/components/partials/Particles.jsx

```diff
@@ -164,17 +161,30 @@ class Particles extends React.Component {
   }
 
   updateDimensions() {
-    this.setState({ width: window.innerWidth });
+    this.setState(() => {
+      const newWidth = window.innerWidth;
+      let newHeight = 421;
+      if (newWidth <= 770) {
+        newHeight = 221;
+      }
+      return { width: newWidth, height: newHeight };
```

Comment:
I think you can also rewrite as: 

```javascript
return {
  width: window.innerWidth
  height: window.innerWidth <= 770 ? 221 : 421// not sure if you need parens for some of this though
}
```

Reply:
One of these days I'll remember that the ternary operator exists but apparently that day is not today (and tomorrow isn't looking too good, either). Updated.